package learn.exception;


// 检查性异常
// 运行时异常
// 错误

// 5个关键字：try catch finally throw throws

public class Demo01 {
    public static void main(String[] args) {

    }



}
